native/src/main/java/org/PDFCreator/PDFGenerator.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.PDFCreator;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import org.apache.pdfbox.pdmodel.font.PDType0Font;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import java.io.File;
+import java.io.IOException;
+
+
+public class PDFGenerator {
+    public static void pdf(String inputFilePath, String replacement, String font_type, int fontSize, int centerX,int centerY,String fontFilePath, String outputFileName) {
```

Comment:
Rather than passing a lot of parameters, can we bundle them into a class and pass an instance of that? The constructor would be much cleaner with that IMO